src/datasources/human-description-api/json/contract-descriptions.json

```diff
@@ -0,0 +1,14 @@
+{
+  "function transfer(address, uint256)": "Send {{tokenValue $1}} to {{address $0}}",
+  "function approve(address, uint256)": "Approve {{tokenValue $1}}",
```

Comment:
Did we make a decision whether we will separate these?

Reply:
I don't think we did. Since the number of functions we support right now is small I suggest to leave this task for later when we add more protocol support and/or extract them to an external source. cc @katspaugh

Reply:
I will move this to the discussion document for v2